src/generate/convert/handle.rs

```
@@ -41,11 +35,7 @@ pub fn convert_handle(ast: &ASTTy, imp: &mut Imports, state: &State, ctx: &Conte
                         match &cond.node {
                             NodeTy::ExpressionType { expr, ty, .. } => {
                                 let expr = Box::from(convert_node(expr, imp, state, ctx)?);
-                                let Some(ty) = ty else {
-                                    let msg = format!("Must have condition, was {cond:?}");
-                                    return Err(Box::from(UnimplementedErr::new(cond, &msg)));
-                                };
-                                let class = Box::from(convert_node(ty, imp, state, ctx)?);
+                                let class = Box::from(ty.as_ref().map_or(Core::Empty, |ty| ty.to_py(imp)));
```

Explicit panic here instead of `Core::Empty`